LICENSE

```diff
@@ -198,4 +198,4 @@
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
\ No newline at end of file
+   limitations under the License.
```

Comment:
In the commit message associated with `make automated style fixes using ruff`, consider listing the fixes that were applied.

For example:
* https://github.com/Slicer/Slicer/pull/7332
* https://github.com/Slicer/Slicer/pull/7647